dependencies.yaml

```diff
@@ -38,6 +38,7 @@ files:
       table: build-system
     includes:
       - build
+      - cuda_version
```

Comment:
This feels like an error. The `cuda_version` dependency list only defines a dependency on the respective `cuda-version` package for _conda_ outputs, but this file output is for `pyproject`. I think we need to remove this.
```suggestion
```

Reply:
Yeah think this is a leftover attempt from trying to fix the CUDA 11.8 error we saw. Agree we can clean this up